deny.toml

```diff
@@ -48,25 +48,21 @@ skip-tree = [
 
     # wait for zcashd and zcash_script to upgrade
     # https://github.com/ZcashFoundation/zcash_script/pulls
```

Comment:
Should we remove these lines now?

Reply:
Nope, we still need `zcashd` to upgrade, unfortunately!

Reply:
https://github.com/zcash/zcash/issues/6648